compiler/rustc_ast_passes/messages.ftl

```diff
@@ -58,6 +56,16 @@ ast_passes_body_in_extern = incorrect `{$kind}` inside `extern` block
 
 ast_passes_bound_in_context = bounds on `type`s in {$ctx} have no effect
 
+ast_passes_c_variadic_associated_function = associated functions cannot have a C-variadic argument
```

Comment:
Technically these are called the "variable arguments" in C, so we may wish to use that name for these cases, specifically
```suggestion
ast_passes_c_variadic_associated_function = associated functions cannot include use C variable argument lists
```